### PR TITLE
A few small tweaks.

### PR DESCRIPTION
**Pull request by pixlwave**
- Use `ShareLink` instead of `InviteFriendsCoordinator`.
- Fix a string in the user menu
- Show the word Notification as the notification's body (it was bold before which was bugging me).

| Notification Before | Notification After |
| - | - |
| IMG_8C233E031F7E-1 | Simulator Screen Shot - iPhone 14 - 2023-04-06 at 11 00 23 |

### CODE REVIEW

File: ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift [hunk 1]
```diff
@@ -122,11 +122,13 @@ struct HomeScreen: View {
                 }
             }
             Section {
-                Button(action: inviteFriends) {
-                    Label(L10n.actionInvite, systemImage: "square.and.arrow.up")
+                if let permalink = context.viewState.invitePermalink {
+                    ShareLink(item: permalink) {
```
Comment (stefanceriu) [inline ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift:126]: Didn't even know this was a thing, good stuff 🙌

**Review by stefanceriu — APPROVED** [review]

File: NSE/Sources/Other/NotificationItemProxy+NSE.swift [hunk 2]
```diff
@@ -73,8 +73,9 @@ extension NotificationItemProxy {
 //            return nil
 //        }
         // For now we can't solve the sender ID nor get the type of message that we are displaying
-        // so we are just going to process all of them as common
-        try await processCommon(senderId: "undefined", roomId: roomId, mediaProvider: mediaProvider)
+        // so we are just going to process all of them as a text notification saying "Notification
```
Comment (Velin92) [inline NSE/Sources/Other/NotificationItemProxy+NSE.swift:76]:
``` suggestion
        // so we are just going to process all of them as a text notification saying "Notification"
```

File: ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift (hunk 1 above)
Comment (pixlwave) [reply ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift:126]: I don't think it was when we first built the menu 😄

Comment (stefanceriu) [reply ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift:126]: Ah 😅

File: NSE/Sources/Other/NotificationItemProxy+NSE.swift (hunk 2 above)
Comment (pixlwave) [reply NSE/Sources/Other/NotificationItemProxy+NSE.swift]: Got distracted before finishing this comment 🙈

**Review by Velin92 — APPROVED** [review]
LGTM

**Pull request merged**